packages/api/src/external/carequality/create-outbound-patient-discovery-req.ts

```diff
@@ -1,44 +1,41 @@
-import { OutboundPatientDiscoveryReq, XCPDGateway } from "@metriport/ihe-gateway-sdk";
 import { Patient as FHIRPatient } from "@medplum/fhirtypes";
+import { OutboundPatientDiscoveryReq, XCPDGateway } from "@metriport/ihe-gateway-sdk";
 import dayjs from "dayjs";
-import { createPurposeOfUse } from "./shared";
+import { HieInitiator } from "../hie/get-hie-initiator";
+import { createPurposeOfUse, getSystemUserName } from "./shared";
 
 export function createOutboundPatientDiscoveryReq({
   patient,
   cxId,
   xcpdGateways,
-  facilityNPI,
-  orgName,
-  orgOid,
+  initiator,
   requestId,
 }: {
   patient: FHIRPatient;
   cxId: string;
   xcpdGateways: XCPDGateway[];
-  facilityNPI: string;
-  orgName: string;
-  orgOid: string;
+  initiator: HieInitiator;
   requestId: string;
 }): OutboundPatientDiscoveryReq {
-  const user = `${orgName} System User`;
+  const user = getSystemUserName(initiator.orgName);
```

Comment:
Sure we should use the Org and not the Facility's name when OBO?

Reply:
Yea, the request is sent by the user of the organization that's acting OBO of that facility.